internal/app/orchestrator/orchestrator_test.go

```diff
@@ -27,6 +29,13 @@ func TestNew(t *testing.T) {
 	}
 	requestMapper := mapper.NewMapper(&config)
 
-	orchestrator := New(context.Background(), log.New("info"), requestMapper, upstreamClient)
+	cacheConfig := bootstrapv1.Cache{
+		Ttl: &duration.Duration{
```

Comment:
shouldnt this be `time.duration`?

Reply:
See https://github.com/envoyproxy/xds-relay/pull/49#discussion_r406479958.